src/app/content/practiceQuestions/components/PracticeQuestionsPopup.spec.tsx

```diff
@@ -26,21 +30,33 @@ jest.mock('react-dom', () => ({
   createPortal: (children: any) => children,
 }));
 
+const mockMatch = {
+  params: {
+    book: { slug: 'book' },
+    page: { slug: 'page' },
+  },
+  route: content,
+};
+
 describe('PracticeQuestions', () => {
   let store: Store;
   let services: ReturnType<typeof createTestServices>;
   let container: HTMLElement;
-  let dispatch: jest.SpyInstance;
 
   beforeEach(() => {
-    store = createTestStore();
-    dispatch = jest.spyOn(store, 'dispatch');
+    store = createTestStore({
+      content: initialContentState,
```

Comment:
this is not needed

Reply:
Are you sure? `store` is used in a `Provider`.